Weather/ForecastIO/weather.15m.py

```diff
@@ -201,6 +207,10 @@ def render_wx():
 
   print '---'
 
+  if 'pressure' in weather_data:
```

Comment:
This is pulling the wrong key value. I would also wrap this below the radar image and keep the next hour summary at the top as it's the most relevant.

Something like this:

```
  print '---'

  if 'next_hour' in weather_data:
    print weather_data['next_hour']

  print '---'

  if 'image' in weather_data:
    if weather_data['image']['encoded']:
      print '| href=' + weather_data['image']['url'] + '| image=' + weather_data['image']['encoded'] 

  print '---'

  if 'week' in weather_data:
    print "\n".join(textwrap.wrap(weather_data['week'], 50))
     print '---'
```